lms/static/sass/_developer.scss

```diff
@@ -286,3 +286,12 @@
     }
   }
 }
+
+//mehrozezahid - TNL-1437
```

Comment:
No it's the convention followed by other developers when adding classes to this file. GitHub user name and ticket followed by some context.